examples/action/XDAI-bridge.xml

```diff
@@ -3,17 +3,17 @@
         <!ENTITY style SYSTEM "shared.css">
         <!ENTITY xdai-bridge.en SYSTEM "xdai-bridge.en.js">
 ]>
-<ts:action xmlns:ts="http://tokenscript.org/2020/06/tokenscript"
+<ts:card type="action" xmlns:ts="http://tokenscript.org/2020/06/tokenscript"
```

Comment:
Shouldn't this be ts:token? Why is action set with the tokenscript namespace?

Reply:
this is an action-only tokenscript. this is it is the root element, there is nowhere else to set the tokenscript namesapce